web/blob/files.go

```diff
@@ -1911,14 +1914,34 @@ var files = map[string]map[string][]byte{
 			0x3d, 0xfd, 0x94, 0xf8, 0x1f, 0xb1, 0xab, 0xd8, 0x1f, 0x3b, 0x19, 0x00, 0x00,
 		},
 
+		"css/.default.css.swp": {
```

Comment:
Because I had a vim running at that time. I already pushed a fix for that and the other web asset issues yesterday:

https://github.com/prometheus/alertmanager/commit/e00f5b183ed432688de9c02961b36df5c0c70797